test/e2e/util.go

```diff
@@ -1067,6 +1081,7 @@ func RunRC(config RCConfig) error {
 				running++
 				for _, v := range FailedContainers(p) {
 					failedContainers = failedContainers + v.restarts
+					containerRestartNodes.Insert(p.Spec.NodeName)
```

Comment:
It might be useful, but in my opinion it should block this PR (it's not that useful).